src/main/java/org/kuali/kfs/vnd/businessobject/VendorDetail.java

```diff
@@ -559,7 +543,7 @@ public String getVendorCommoditiesAsString() {
             } else {
                 first = false;
             }
-            sb.append(vcc.getCommodityCode().getCommodityDescription());
+            sb.append(vcc.getCommodityCode().getPurchasingCommodityCode());
```

Comment:
I believe this is a line that we customized for the KFSPTS-3487 enhancement. This should not be changed back to the KualiCo version of the line unless the customization is no longer needed.

Reply:
good catch, I shall change it back